### ISSUE TITLE
Text classification predictionEnginePool Predict result not calibrated.

### ISSUE THREAD

**Issue by zewditu** [dotnet/machinelearning#6516]
**System Information (please complete the following information):**
 - OS & Version: [e.g. Windows 10] 
 - ML.NET Version: [e.g. ML.NET v1.5.5]
 - .NET Version: [e.g. .NET 5.0]

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots, Code, Sample Projects**
![image](https://user-images.githubusercontent.com/36615490/205402950-724be94e-0ea2-4531-b75e-a8be8b43241c.png)

**Additional context**

**Comment by michaelgsharp** [comment]
@zewditu this seems like an AutoML issue. We probably don't want the calibration in by default do we @luisquintanilla in the framework? That way the can calibrate it if they want it? Or do they always want it calibrated?

**Comment by luisquintanilla** [comment]
> @zewditu this seems like an AutoML issue. We probably don't want the calibration in by default do we @luisquintanilla in the framework? That way the can calibrate it if they want it? Or do they always want it calibrated?

Correct. I think this is something we might want to solve in tooling by appending a calibrator to the pipeline like other algorithms that don't do it by default unless there's a good reason for us to do it in the framework.

@zewditu here is a sample which appends a callibrator to a pipeline: https://gist.github.com/luisquintanilla/77664adc3f0e920648954244893a6bf8

**Comment by luisquintanilla** [comment]
Moving this issue to the Model Builder repo.
